pkg/spdk/engine.go

```diff
@@ -61,7 +61,8 @@ type Engine struct {
 	Head        *api.Lvol
 	SnapshotMap map[string]*api.Lvol
 
-	IsRestoring bool
+	IsRestoring           bool
+	RestoringSnapshotName string
```

Comment:
For a DR volume, is this the final snapshot name, or the snapshot name of the upcoming incremental restore?

Reply:
Final and upcoming, only this snapshot name will be used during the restore.

Reply:
Does the engine need to cache this? At least for now, it is used in `Engine. BackupRestore` only

Reply:
We will include e.RestoringSnapshotName in the BackupRestoreRequest to ensure the snapshot name is consistent.

Reply:
OK. It make sure DR volume snapshot name keeps unchanged